foundation-extensions/profile/profile-api-references.md

```diff
@@ -338,6 +379,117 @@ private void handleCallback(NSDictionary content, NSError error)
 {% endtabs %}
 
 
+## registerExtension
+
+Registers the Profile extension with the Mobile Core extension.
+
+{% tabs %}
+{% tab title="Android" %}
+
+### Java
+
+**Syntax**
+
+```java
+public static void registerExtension()
+```
+
+**Example**
+```java
+import com.adobe.marketing.mobile.UserProfile
+
+...
+
+UserProfile.registerExtension();
+```
+
+{% endtab %}
+
+{% tab title="iOS (AEP 3.x)" %}
+
+In iOS, the registration occurs by passing UserProfile extension to the [MobileCore.registerExtensions API](../mobile-core/mobile-core-api-reference.md#registerextension-s).
+
+### Swift
+
+**Syntax**
+
+```swift
+static func registerExtensions(_ extensions: [NSObject.Type], 
+                               _ completion: (() -> Void)? = nil)
+```
+
+**Example**
+
+```swift
+import AEPUserProfile
+
+...
+MobileCore.registerExtensions([UserProfile.self])
+```
```

Comment:
For consistency with the ACP tab, can we also include the didFinishLaunchingWithOptions in these AEP samples?

Reply:
Sure, I will update the AEP tab.